### PR TITLE
fix: run bin abort signal

### LINKED ISSUE #735: framebinprovider doesn't check if process is already killed
```
error: Uncaught TypeError: Child process has already terminated.
    this.env.signal.addEventListener("abort", () => process.kill("SIGINT"))
                                                            ^
    at ChildProcess.kill (ext:runtime/40_process.js:321:13)
    at AbortSignal.<anonymous> (file:///github/workspace/providers/frame/FrameBinProvider.ts:48:61)
    at innerInvokeEventListeners (ext:deno_web/02_event.js:783:7)
    at invokeEventListeners (ext:deno_web/02_event.js:823:5)
    at dispatch (ext:deno_web/02_event.js:692:9)
    at AbortSignal.dispatchEvent (ext:deno_web/02_event.js:1082:12)
    at AbortSignal.[[[signalAbort]]] (ext:deno_web/03_abort_signal.js:92:10)
    at AbortController.abort (ext:deno_web/03_abort_signal.js:165:30)
    at onReady (file:///github/workspace/main.ts:77:16)
    at async Object.onListen (file:///github/workspace/main.ts:63:7)
```

### PR DESCRIPTION
**Pull request by ryanleecode**
fixes #735

### CODE REVIEW

File: providers/frame/FrameBinProvider.ts [hunk 1]
```diff
@@ -43,10 +43,10 @@ export abstract class FrameBinProvider extends FrameProxyProvider {
       args,
       stdout: "piped",
       stderr: "piped",
+      signal: this.env.signal,
```
Comment (ryanleecode) [inline providers/frame/FrameBinProvider.ts:46]: not sure if this requires a `.bind(this)`

Comment (tjjfvi) [reply providers/frame/FrameBinProvider.ts:46]: It's not a function, so no

**Comment by tjjfvi** [comment]
Ah, good catch. Can you make sure this is done everywhere else we use `Deno.Command`? (Or is this the only relevant place?)

**Comment by ryanleecode** [comment]
should be good now

**Comment by harrysolovay** [comment]
It strikes me that this change should (possibly) be based off of #777.

**Review by tjjfvi — APPROVED** [review]

**Pull request merged**